src/autolabel/models/vllm.py

```diff
@@ -0,0 +1,93 @@
+import json
+import os
+import requests
+import logging
+from typing import List, Optional, Tuple
+from time import time
+from vllm import LLM, SamplingParams
```

Comment:
We might want t add this to autolabel requirements as well.

Reply:
not adding this to requirements as this is experimental. but still fixed how this would be used